delfin/api/v1/alert.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 from oslo_log import log
+from pyasn1.type.univ import OctetString
```

Comment:
pyasn1 is a standard lib of python?

Reply:
yes, it is an ASN library for python. Also pysnmp internally make use of it for validations

Reply:
got it